web/docs/tutorial/02-project-structure.md

```diff
@@ -26,14 +28,43 @@ After creating a new Wasp project, you'll get a file structure that looks like t
 
 ```
 
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+The default project uses JavaScript. To use TypeScript, you must manually rename the file
+`src/MainPage.jsx` to `src/MainPage.tsx`.
+
+No updates to the `main.wasp` file are necessary - it stays the same regardless of the language you use.
+
+After creating a new Wasp project and renaming the `src/MainPage.jsx` file, your project should look like this:
+
+```
+.
+├── .gitignore
+├── main.wasp          # Your Wasp code goes here.
+├── schema.prisma      # Your Prisma schema goes here.
+├── package.json       # Your dependencies and project info go here.
+├── public             # Your static files (e.g., images, favicon) go here.
+├── src                # Your source code (TS/JS/CSS/HTML) goes here.
+│   ├── Main.css
+│   ├── MainPage.tsx
```

Comment:
```suggestion
// highlight-next-line
│   ├── MainPage.tsx
```
And you need some language for the code block e.g. `c` for the highlight to work